gr4vy/gr4vy_client.py

```diff
@@ -162,9 +168,40 @@ def _thumbprint(self, jwk: dict) -> str:
         return self._b64e(digest.finalize())
 
     def generate_embed_token(self, embed_data, checkout_session_id=None):
-        token = self.generate_token(embed_data=embed_data, checkout_session_id=checkout_session_id)
+        token = self.generate_token(
+            embed_data=embed_data, checkout_session_id=checkout_session_id
+        )
         return token
 
+    def verify_webhook(
+        self,
+        secret: str,
+        payload: str,
+        signature_header: typing.Optional[str],
+        timestamp_header: typing.Optional[str],
```

Comment:
Add default?
```suggestion
        timestamp_header: typing.Optional[str] = None,
```

Reply:
We actually need it to calculate the expected signature. I made these both optional so the usage is easier as the caller can just default to `None` when searching for the header.